setup.py

```diff
@@ -190,7 +190,7 @@ def run(self):
             "Running flake8 ...", [sys.executable, "-m", "flake8", f"{here}/slack_sdk"]
         )
 
-        target = self.test_target.replace("tests/", "")
+        target = self.test_target.replace("tests/", "", 1)
```

Comment:
To avoid replacing `tests/` twice in `tests/slack_sdk_tests/`. In another pull request, I'll rename the test packages.